### PR TITLE
change aft summary from counter64 to uint32

### LINKED ISSUE #1010: AFT Summary model, incorrect type
PR#997 (https://github.com/openconfig/public/pull/997) introduced AFT summary 'counters', however they are defined using the type counter64 (which are monotonically incrementing). 

```
        |  +--ro oc-aftsummary:aft-summaries
        |     +--ro oc-aftsummary:ipv4-unicast
        |     |  +--ro oc-aftsummary:protocols
        |     |     +--ro oc-aftsummary:protocol* [origin-protocol]
        |     |        +--ro oc-aftsummary:origin-protocol    -> ../state/origin-protocol
        |     |        +--ro oc-aftsummary:state
        |     |           +--ro oc-aftsummary:origin-protocol?   identityref
        |     |           +--ro oc-aftsummary:counters
        |     |              +--ro oc-aftsummary:aft-entries?   oc-yang-types:counter64
        |     +--ro oc-aftsummary:ipv6-unicast
        |        +--ro oc-aftsummary:protocols
        |           +--ro oc-aftsummary:protocol* [origin-protocol]
        |              +--ro oc-aftsummary:origin-protocol    -> ../state/origin-protocol
        |              +--ro oc-aftsummary:state
        |                 +--ro oc-aftsummary:origin-protocol?   identityref
        |                 +--ro oc-aftsummary:counters
        |                    +--ro oc-aftsummary:aft-entries?   oc-yang-types:counter64
```

The type for these should probably be changed to something like gauge32 or uint32. BGP reports similar data using uint32.

```
        |     |  |     |     |  +--ro prefixes
        |     |  |     |     |     +--ro received?              uint32
        |     |  |     |     |     +--ro received-pre-policy?   uint32
        |     |  |     |     |     +--ro sent?                  uint32
        |     |  |     |     |     +--ro installed?             uint32 
        ```

### PR DESCRIPTION
**Pull request by dplore**
Fixes #1010

### CODE REVIEW

**Comment by rolandphung** [comment]
Should this be considered not backwards compatible?

**Comment by dplore** [comment]
> Should this be considered not backwards compatible?

Yes.  Note however that we've [asserted](https://github.com/openconfig/public/blob/master/doc/releases.md) that pre v1.0.0 models may have breaking changes that do not cause the OpenConfig release tag to be incremented.

File: release/models/aft/openconfig-aft-summary.yang [hunk 1]
```diff
@@ -49,7 +54,7 @@ module openconfig-aft-summary {
       leaf aft-entries {
         description
           "Total number of entries in the aft.";
-        type oc-yang-types:counter64;
+        type uint32;
```
Comment (robshakir) [inline release/models/aft/openconfig-aft-summary.yang:57]: Suggest to change this to /64.

**Review by robshakir — APPROVED** [review]

File: release/models/aft/openconfig-aft-summary.yang (hunk 1 above)
Comment (dplore) [reply release/models/aft/openconfig-aft-summary.yang]: done

**Pull request merged**